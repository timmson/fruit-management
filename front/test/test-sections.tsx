import React from "react"
import renderer from "react-test-renderer"
import Context from "../src/context"
import Sections from "../src/sections"

describe("Sections should", () => {

    test("return sections", () => {
        const context = []

        const component = renderer.create(
            <Context.Provider value={context}>
                <Sections/>
            </Context.Provider>
        )

        expect(component.toJSON()).toMatchSnapshot()

        component.unmount()
    })

})