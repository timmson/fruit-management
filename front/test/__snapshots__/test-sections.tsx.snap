// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sections should return sections 1`] = `
<table
  className="plain"
>
  <tbody>
    <tr>
      <td
        className="pt-2 text-center align-bottom"
        onClick={[Function]}
        style={
          {
            "width": "15%",
          }
        }
      >
        <p
          className="av"
        >
          <img
            alt="project"
            src="img/admin_project.gif"
          />
          <br />
          Проекты
        </p>
      </td>
      <td
        className="pt-2 text-center align-bottom"
        onClick={[Function]}
        style={
          {
            "width": "15%",
          }
        }
      >
        <p
          className="av"
        >
          <img
            alt="task"
            src="img/admin_task.gif"
          />
          <br />
          Задачи
        </p>
      </td>
      <td
        className="pt-2 text-center align-bottom"
        onClick={[Function]}
        style={
          {
            "width": "15%",
          }
        }
      >
        <p
          className="av"
        >
          <img
            alt="agile"
            src="img/admin_agile.gif"
          />
          <br />
          Agile
        </p>
      </td>
      <td
        className="pt-2 text-center align-bottom"
        onClick={[Function]}
        style={
          {
            "width": "15%",
          }
        }
      >
        <p
          className="av"
        >
          <img
            alt="plan"
            src="img/admin_plan.gif"
          />
          <br />
          Планирование
        </p>
      </td>
      <td
        className="pt-2 text-center align-bottom"
        onClick={[Function]}
        style={
          {
            "width": "15%",
          }
        }
      >
        <p
          className="av"
        >
          <img
            alt="user"
            src="img/admin_user.gif"
          />
          <br />
          Пользователи
        </p>
      </td>
      <td
        className="pt-2 text-center align-bottom"
        onClick={[Function]}
        style={
          {
            "width": "15%",
          }
        }
      >
        <p
          className="av"
        >
          <img
            alt="export"
            src="img/admin_export.gif"
          />
          <br />
          Выгрузка
        </p>
      </td>
    </tr>
  </tbody>
</table>
`;
